### PR TITLE
Adding a comparison plot on two tags for JetMET JEC PI

### PR DESCRIPTION
**Pull request by yuanchao**
#### PR description:

Requested during the AlCa meeting on May 8, adding comparison plots for two different tags JetCorrector payload inspector. Correction level and jet parameters can be set through the input parameter interface of PI.

#### PR validation:

Local tests with getPayloadData.py script.

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

This is not a backport.

### CODE REVIEW

File: CondCore/JetMETPlugins/plugins/JetCorrector_PayloadInspector.cc [hunk 1]
```diff
@@ -82,6 +82,148 @@ namespace {
...
+
+    double par_JetPt = 100.;
+    double par_JetEta = 0.;
+    double par_JetA = 0.5;
+    double par_Rho = 20.;
+    double par_JetPhi = 0.;
+    double par_JetE = 150.;
+
+    // Default values will be used if no input parameters
+    //auto paramValues = cond::payloadInspector::PlotBase::inputParamValues();
```
Comment (tvami) [inline CondCore/JetMETPlugins/plugins/JetCorrector_PayloadInspector.cc:146]: Please remove commented out code

File: CondCore/JetMETPlugins/plugins/JetCorrector_PayloadInspector.cc [hunk 2]
```diff
@@ -102,9 +244,11 @@ namespace {
 
     bool fill() override {
       double par_JetPt = 100.;
-      double par_JetEta = 0.;
+      //      double par_JetEta = 0.;
       double par_Rho = 20.;
-      double par_JetA = 0.5;
+      //      double par_JetPhi = 0.;
+      //      double par_JetE = 150.;
+      //      double par_JetA = 0.5;
```
Comment (tvami) [inline CondCore/JetMETPlugins/plugins/JetCorrector_PayloadInspector.cc:251]: Please remove commented out code

File: CondCore/JetMETPlugins/plugins/JetCorrector_PayloadInspector.cc [hunk 3]
```diff
@@ -470,6 +433,153 @@ namespace {
...
+      leg_eta->SetTextFont(42);
+
+      auto tag1 = PlotBase::getTag<0>();
+      auto iov1 = tag1.iovs.front();
+
+      auto tag2 = PlotBase::getTag<1>();
+      auto iov2 = tag2.iovs.front();
+
+      std::shared_ptr<JetCorrectorParametersCollection> payload1 = fetchPayload(std::get<1>(iov1));
+      //unsigned int run1 = std::get<0>(iov1);
```
Comment (tvami) [inline CondCore/JetMETPlugins/plugins/JetCorrector_PayloadInspector.cc:481]: Please remove commented out code

File: CondCore/JetMETPlugins/plugins/JetCorrector_PayloadInspector.cc [hunk 4]
```diff
@@ -470,6 +433,153 @@ namespace {
...
+      std::string tagname1 = tag1.name;
+
+      std::shared_ptr<JetCorrectorParametersCollection> payload2 = fetchPayload(std::get<1>(iov2));
+      //unsigned int run2 = std::get<0>(iov2);
```
Comment (tvami) [inline CondCore/JetMETPlugins/plugins/JetCorrector_PayloadInspector.cc:485]: Please remove commented out code

**Review by tvami — COMMENTED** [review]
Please remove commented out code

File: CondCore/JetMETPlugins/plugins/JetCorrector_PayloadInspector.cc (hunk 1 above)
Comment (yuanchao) [reply CondCore/JetMETPlugins/plugins/JetCorrector_PayloadInspector.cc]: Done in the latest commit.

**Comment by tvami** [comment]
@cmsbuild , please test

**Comment by tvami** [comment]
+1

**Comment by rappoccio** [comment]
+1

**Pull request merged**